webroot/js/participantSummary.js

```diff
@@ -123,7 +164,23 @@ $(document).ready(() => {
 
         url = '/api/partner_config/get';
         doApiCallWithCallback('GET', url, (r) => { loadOptoutWebhooksCallback(r, site.name) }, (err) => { participantSummaryErrorHandler(err, '#webhooksErrorOutput') });
-    
+
+        url = `/api/sharing/keysets/related?site_id=${site.id}`;
+        doApiCallWithCallback('GET', url, (r) => { loadRelatedKeysetsCallback(r, site.id, site.clientTypes) }, (err) => { participantSummaryErrorHandler(err, '#relatedKeysetsErrorOutput') });
         $('.section').show();
     });
+
+    $('#doRotateKeysets').on('click', () => {
+        var keysets = $('#relatedKeysetsStandardOutput').text();
+        const ja = JSON.parse(keysets);
+        var rotateKeysetsMessage = '';
+        ja.forEach((keyset) => {
+            var url = `/api/key/rotate_keyset_key?min_age_seconds=3600&keyset_id=${keyset.keyset_id}&force=true`;
+            doApiCallWithCallback(
```

Comment:
See result of this in the PR description screenshot. 

Each time a keyset is rotated, the rotated keyset information will show on the UI. cc: @vishalegbert-ttd